### PR TITLE
fix: BFF peer file URLS

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** [ A short summary of what this PR does and why ]

JIRA: [AU-1579](https://2u-internal.atlassian.net/browse/AU-1579)

**What changed?**

- Fixed URL generations for peer file uploads.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

Peer grading workflow.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/__init__.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 Initialization Information for Open Assessment Module
 """
 
-__version__ = '6.0.8'
+__version__ = '6.0.9'
```
Comment (leangseu-edx) [inline openassessment/__init__.py:5]: 6.0.10

**Review by leangseu-edx — APPROVED** [review]
